### PR TITLE
feat(element-template-tab): indicate element template not found

### PR DESCRIPTION
**Pull request by philippfromme**
image

__Which issue does this PR address?__

Related to camunda/camunda-modeler#1891

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: lib/provider/camunda/element-templates/parts/DescriptionProps.js [hunk 1]
```diff
@@ -164,4 +175,30 @@ function getEventDefinitionType(businessObject) {
...
+      '<br />' +
+      'Unlink in order to access the data.', { templateId: templateId }
+    ) +
+    '</p>' +
+    '<p style="text-align: right;">' +
+    '<a class="bpp-entry-link bpp-entry-link-button">Unlink</a>' +
+    '</p>'
+  );
+
+  domQuery('.bpp-entry-link', html).addEventListener('click', function() {
```
Comment (pinussilvestrus) [inline lib/provider/camunda/element-templates/parts/DescriptionProps.js:197]: I like that we don't try to handle this via `data-action` (if it's possible with that at all) 😄  That's fine for me.

**Review by pinussilvestrus — APPROVED** [review]
Changes look good to me. Is there an additional design review needed?

File: lib/provider/camunda/element-templates/parts/DescriptionProps.js (hunk 1 above)
Comment (barmac) [reply lib/provider/camunda/element-templates/parts/DescriptionProps.js:197]: At least we can avoid the `set` hack 👍

**Comment by andreasgeier** [comment]
What I can see in the screenshot above looks good to me. No dedicated UI review needed, I guess.

**Pull request merged**